src/Features/Core/Portable/NavigateTo/NavigateToMatchKind.cs

```diff
@@ -10,5 +10,11 @@ internal enum NavigateToMatchKind
         Substring = 2,
         Regular = 3,
         None = 4,
+        CamelCaseExact = 5,
```

Comment:
I see. it used to different. but now we basically made all same since we need to pass info we used to throw away back to editor. I think we should just remove this enum and use PatternMatchKind all the way.